lib/interpolation.ts

```diff
@@ -1,5 +1,12 @@
-// @ts-nocheck
-const helpers = require('./interpolationHelpers');
+import * as helpers from './interpolationHelpers';
+
+interface interpolationDataType {
```

Comment:
I think this type could just be `InterpolationData`. Also just for consistency, I think this could be a `type` and not an `interface`